### DISCUSSION TITLE
Some questions from akkypat

### DISCUSSION THREAD

**Discussion by cslotboom** [cslotboom/planesections#19] [Q&A]
@akkypat 
I'm following up on your question here

Hello!
I am writing a calculator for solving problems in mechanics. I want to compare your values with mine. I can't find the information in the documentation information.
1)Can you tell me how to display the values on the graph and how to output the y values for the values of some x ?
2) Is it possible to shade the area under the graph?
An example is attached.
3) How to plot the result of the reactions of the supports Ra and Rb
![image](https://github.com/user-attachments/assets/ca2f4b2d-c30d-4cf2-a01c-396933a4d2c0)

**Comment by cslotboom** [comment]
Thanks for your questions. To answer
1.) You can set the tag plotForceValue=True in the function plotBeamDiagram
2.) Right now it isn't - I can consider this as a development option though. It would help if you can be specific with what you're looking for. Do you want a solid colour or lines as shown?
3.) Right now you can't plot reactions. I could also add this as a feature.

**Reply by cslotboom** [reply]
Also, note some of the documentation is broken. This function should have documentation and required an update on my end:
https://planesections.readthedocs.io/en/latest/rst/diagram-plotBeamDiagram.html

**Reply by akkypat** [reply]
Hi!
Thank you very much for your reply. 
1. I used this parameter, but the values appeared only for the input data. Calculated data for moments and forces are not shown. Ex2.
![Ex2](https://github.com/user-attachments/assets/ce4a789e-2a33-4a1d-834e-0beefbcd1e4a)

2. I would like the lines as in the picture. This is how the plot has always been designated in our university. They can be drawn using the matplotlib module, but this requires x and y values. I could not find information in the documentation on how to get them.
3. It would be very cool!
All the best to you

**Reply by cslotboom** [reply]
Hey, If you check out the plot functions, there is a flag which will turn on the values for item 1! I will fix the readthedocs website when I get a chance. For 2, you can get the x/y points from the matplotlib line object - it's returned from the plot function. It's not ideal but from there you could implement something that draws the lines. I'm caught up in other development efforts right now unfortunately, so will be a little bit before I can sink time into this, but hopefully that helps for now.
